lib/9pfs/9pfs_vnops.c

```diff
@@ -1011,8 +1011,18 @@ static int uk_9pfs_symlink(struct vnode *dvp, const char *op, const char *np)
 	return 0;
 }
 
+static int uk_9pfs_ioctl(struct vnode *dvp, struct vfscore_file *fp,
+			 unsigned long com, void *data)
+{
+	switch (com) {
+	case TIOCGWINSZ:
+		return EBADF;
```

Comment:
Is there any reason we don't return `ENOTTY` here? I might be missing something, but  that seems like the proper errno code.

Reply:
No reason, just ignorance :dancers:.